src/runtime/objects/activation.zig

```diff
@@ -186,7 +187,10 @@ pub const Activation = extern struct {
     pub fn getAssignableSlotValue(self: Activation.Ptr, slot: Slot) *GenericValue {
         std.debug.assert(slot.isAssignable());
 
-        const offset: usize = @intCast(slot.value.asUnsignedInteger());
+        const offset_int = slot.value.asUnsignedInteger();
+        std.debug.assert(!exceedsBoundsOf(offset_int, usize));
```

Comment:
@sin-ack I'm not sure about this one. The function doesn't really return an error and I didn't want to change too much right now.

Reply:
This is good enough for now, we can make it return a user-facing error later.